tests/framework/CodeInterfaceTests/test_generic_interface.xml

```diff
@@ -27,6 +27,7 @@
     <Input name="one.xml" type="">one.xml</Input>
     <Input name="inp.two" type="">inp.two</Input>
     <Input name="mesh" type="">dummy.e</Input>
+    <Input name="a_dummy_file_for_subdirectory" type="" subDirectory="testSubDirectory">dummy_file_for_subdirectory.dummy</Input>
```

Comment:
How exactly does this work? Is this file generated or an input file? Or does it get copied from the working directory into some subdirectory?